CxScaResolver/azure-pipelines.yml

```diff
@@ -0,0 +1,30 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
```

Comment:
Let's replace this comment with the documentation link we have on the other resolver examples

Reply:
Done